apps/roboshield/next.config.mjs

```diff
@@ -0,0 +1,29 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+  transpilePackages: ["@commons-ui/core", "@commons-ui/next"],
+  reactStrictMode: false,
+  output: "standalone",
+  webpack: (config) => {
+    config.module.rules.push(
+      {
+        test: /\.svg$/i,
+        type: "asset",
+        resourceQuery: /url/, // *.svg?url
+      },
+      {
+        test: /\.svg$/i,
+        issuer: /\.[jt]sx?$/,
+        resourceQuery: { not: [/url/] }, // exclude react component if *.svg?url
+        use: ["@svgr/webpack"],
+      },
+      {
+        test: /\.md$/,
+        loader: "frontmatter-markdown-loader",
+      },
```

Comment:
Are we using Markdown files anywhere?

Reply:
We are not using Markdown anywhere, I've removed `frontmatter-markdown-loader`